src/Biblys/Isbn/Isbn.php

```diff
@@ -311,9 +332,19 @@ private function calculateChecksum($format = 'EAN')
                 $sum = 'X';
             }
         } else {
-            $code = $this->getProduct().$this->getCountry().$this->getPublisher().$this->getPublication();
-            $c = str_split($code);
-            $sum = (($c[1] + $c[3] + $c[5] + $c[7] + $c[9] + $c[11]) * 3) + ($c[0] + $c[2] + $c[4] + $c[6] + $c[8] + $c[10]);
+            $code = $this->getGtin14Prefix().$this->getProduct().$this->getCountry().$this->getPublisher().$this->getPublication();
+            $c = array_reverse(str_split($code));
+
+            foreach ($c as $k => $v) {
+                if ($k & 1) { // If current array key is odd
+                    $sum += $v;
+                } else { // If current array key is even
+                    $sum += $v * 3;
+                }
+            }
+
+            \var_dump($sum);
```

Comment:
You forgot a `var_dump` here :)

Reply:
Oh ferchrissakes, I’d forget my brain if it weren’t screwed on to the inside of my head!

Reply:
I also forgot to add a PHPDoc line for the new parameter `$prefix` to the `format()` function (fixed now).